src/Examples/Issues/Issue855Proto2CSharp.Generated.cs

```diff
@@ -0,0 +1,132 @@
+﻿// <auto-generated>
```

Comment:
I guess this file should be git-ignored and not to be added to source control, as it is generated file.
Right?